### PR TITLE
:bug: get the correct plotly_domain, not the "session"'s

### PR DESCRIPTION
**Pull request by chriddyp**

### CODE REVIEW

**Comment by chriddyp** [comment]
cc @theengineear

File: plotly/offline/offline.py [hunk 1]
```diff
@@ -129,8 +130,10 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
     config['linkText'] = link_text
     jconfig = json.dumps(config)
 
-    plotly_platform_url = session.get_session_config().get('plotly_domain',
-                                                           'https://plot.ly')
+    # TODO: The get_config 'source of truth' should
+    # really be somewhere other than plotly.plotly
```
Comment (theengineear) [inline plotly/offline/offline.py]: Mind tagging this with an issue in this repo? :hamster:

**Comment by theengineear** [comment]
Loving the commit emojis :)

Non-blocking --> :dancer:

File: plotly/offline/offline.py (hunk 1 above)
Comment (chriddyp) [inline plotly/offline/offline.py]: https://github.com/plotly/plotly.py/issues/364

**Pull request merged**